python_modules/libraries/dagster-aws/dagster_aws/pipes/clients/ecs.py

```diff
@@ -69,18 +69,26 @@ def __init__(
     def _is_dagster_maintained(cls) -> bool:
         return True
 
+    @public
     def run(
         self,
         *,
         context: OpExecutionContext,
         task_definition: str,
-        count: int,
+        count: int = 1,
         extras: Optional[Dict[str, Any]] = None,
         **params: Unpack[RunECSTaskParams],
     ) -> PipesClientCompletedInvocation:
         """Start a ECS task, enriched with the pipes protocol.
 
-        See also: `AWS API Documentation <https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RunTask.html>`_
+        Args:
+            context (OpExecutionContext): The Dagster op context.
+            task_definition (str): ECS task definition to use.
+            count (int): The number of tasks to run.
```

Comment:
Is the `= 1` intentional, we may want to include a mention of the defaults here

Reply:
Determined in DM that this is not necessary, but will be exposed in `**params`

Reply:
turns out this parameter is optional, removed it